hms-app/src/components/admin-tools/HackathonForm.tsx

```diff
@@ -30,28 +30,26 @@ const useStyles = createStyles((theme) => ({
   },
 }))
 
-function NewHackathon() {
+type IProps = { context: string; hackathonID: string | null }
+
+function HackathonForm(props: IProps) {
```

Comment:
Might be able to make a better name for this as well, but it performs two functions to both create and edit hackathons.  I guess you are using `Form` in the sense of a `Form` on a website that allows data entry... maybe thats ok then :thinking: